materials/01-what-is-vc.qmd

```diff
@@ -2,12 +2,63 @@
+
+## 1.2 Why Version Control?
+
+Version control is the lab notebook of the digital world: it's what professionals use to keep track of what they've done and to collaborate with other people.
+
+Version control is better than sharing files back and forth because:
+
+-   Nothing that is committed to version control is ever lost, unless you work really, really hard at it. Since all old versions of files are saved, it's always possible to go back in time to see exactly who wrote what on a particular day, or what version of a program was used to generate a particular set of results.
+
+-   As we have this record of who made what changes when, we know who to ask if we have questions later on, and, if needed, revert to a previous version, much like the "undo" feature in an editor.
+
+-   When several people collaborate in the same project, it's possible to accidentally overlook or overwrite someone's changes. The version control system automatically notifies users whenever there's a conflict between one person's work and another's.
+
+::: callout-note
+Teams are not the only ones to benefit from version control: lone researchers can benefit immensely. Keeping a record of what was changed, when, and why is extremely useful for all researchers if they ever need to come back to the project later on (e.g., a year later, when memory has faded).
+:::
+
+## 1.3 What is a Version Control Repository?
+
+In version control systems, a repository is a data structure that stores metadata for a set of files or directory structure. Some of the metadata that a repository contains includes, among other things, a historical record of changes in the repository, a set of commit objects, and a set of references to commit objects, called heads. We will go into more detail on these during the workshop.
+
+Essentially, a version control repository keeps track of a set of files in a project. Using the WES analysis example above, you and your colleagues may choose to structure your version control project like this:
+
+```         
+├──wes-analysis
+|  ├── data
+|  │   ├── wes-results.csv
+|  │   │   ├── wes-results-2019.csv
+|  │   │   ├── wes-results-2020.csv
+|  │   │   ├── wes-results-2021.csv
+|  │   |   |── wes-results-2022.csv
+|  ├── doc
+|  ├── ref
+|  │   ├── references
+|  │   ├── images
+|  ├── plots
+|  ├── code
+|  ├── .git
 ```
 
-## 
+Which is essentially a regular project folder containing subfolders to organize your work, but with an additional `.git` folder in which to house the metadata.
```

Comment:
I would add a disclaimer or something here - this makes it sound like they have to manually create the `.git` folder themselves, whereas that will be handled by either git or GitHub, depending on how you set it up.